### ISSUE TITLE
WordPress Version?

### ISSUE THREAD

**Issue by ghost** [jalxob/cool-kitten#26]
I love your framework! But I was wondering if you plan on trying to port it to a WordPress Theme Framework?

If so, that would be awesome!

**Comment by chrisryancarter** [comment]
I adapted it for wordpress. I could post it if @jalxob doesn't mind.

**Comment by jalxob** [comment]
@atonechris Do you think we can discuss it on private? I have a good idea :)

**Comment by chrisryancarter** [comment]
Sure! chris {at} clergycreative.com

**Comment by Joepmuijres** [comment]
@atonechris @jalxob . I am very curious to the wordpress framework. Guess this is the one i've been looking for!!! Can i try it? I would love to try/use it for a non-commercial musicband-website. Let me know!
Thanks

**Comment by marilynwo** [comment]
Yes, thank you for the framework, how do we use it in Wordpress?

**Comment by jcuenin** [comment]
Hello @jalxob , any news regarding the Wordpress version of this html/css framework? I have to create a parallax theme for a client and would love to have a WordPress version of it.
